x-pack/plugins/security_solution/public/detections/components/rules/related_integrations/use_integrations.test.tsx

```diff
@@ -5,14 +5,13 @@
  * 2.0.
  */
 
-import React from 'react';
-import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { renderHook, cleanup } from '@testing-library/react-hooks';
 
-import { useInstalledIntegrations } from './use_installed_integrations';
+import { useIntegrations } from './use_integrations';
```

Comment:
`describe('useInstalledIntegrations' ...` on line 19 
should now also be renamed to 
`describe('useIntegrations' ...`